docs/test-pages/TCD-008.adoc

```diff
@@ -0,0 +1,36 @@
+= TCD-008
+
+== Purpose
+
+Verifying the primitive types and their working. This case is verifying the optional primitive fields without adding any values to them.
+
+[NOTE]
+This is a reverse engineered test case.
+
+=== Tested requirements
+
+REQ-TYPE-001
+
+=== Used JSL model
+
+xref:resources/Primitives.jsl[Primitives.jsl]
+
+== Test scenario
+
+. Create and save MyEntityWithOptionalFields without adding any attribute value.
+
+. Read from the database the instance of MyEntityWithOptionalFields that saved in the previous step (entity1).
```

Comment:
"Re-read the instance of MyEntityWithOptionalFields that has been saved in the previous step (entity1)."